### ISSUE TITLE
Wrong disabled class on years with min-date, max-date settings

### ISSUE THREAD

**Issue by csimpi** [indrimuska/angular-moment-picker#270]
1999 is disabled despite the date range:

![image](https://user-images.githubusercontent.com/6192067/45382645-bd0d1e00-b5bd-11e8-96a2-1bf215150399.png)

Prev page, 1999 is enabled but 1989 is disabled. It seems every first and last year is disabled without any reason:

![image](https://user-images.githubusercontent.com/6192067/45382673-d2824800-b5bd-11e8-906a-817eacfc29a4.png)

**Comment by csimpi** [comment]
Update, I changed this line
`!selectable || [0, 11].indexOf(y) >= 0 ? 'disabled' : utility_1.isValidMoment(this.$ctrl.$modelValue) && year.isSame(this.$ctrl.$modelValue, 'year') ? 'selected' : ''`
to this:
`!selectable ? 'disabled' : utility_1.isValidMoment(this.$ctrl.$modelValue) && year.isSame(this.$ctrl.$modelValue, 'year') ? 'selected' : ''`

in the `DecadeView.prototype.render()` runction

Basically I removed this condition:
`|| [0, 11].indexOf(y) >= 0`

I don't know if it has any goal let me know if I misunderstood something.
Thx
